SpatialGDK/Source/SpatialGDK/Public/SpatialGDKSettings.h

```diff
@@ -238,10 +238,14 @@ class SPATIALGDK_API USpatialGDKSettings : public UObject
 	UPROPERTY(EditAnywhere, Config, Category = "Load Balancing", meta = (EditCondition = "bEnableUnrealLoadBalancer"))
 	FWorkerType LoadBalancingWorkerType;
 
+	/** RPC ring buffers is enabled when either the matching setting is set, or zoning is enabled */
```

Comment:
Nit would prefer that we saying load balancing is enabled. Zoning is a badly defined term.